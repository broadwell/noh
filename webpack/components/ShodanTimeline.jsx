import React from "react";

// the shodan in here need to be abstracted into a reusable component
const ShodanTimeline = () => (
  <div className="shodan-map">
  {" "}
    <div
      className="shodan-map__item"
      style={{ left: "0%", width: "3%", height: "10%" }}
      data-tooltip="Nanori"
    />
    <div
      className="shodan-map__item"
      style={{ left: "3%", width: "7%", height: "15%" }}
      data-tooltip="Mondo"
    />
    <div
      className="shodan-map__item"
      style={{ left: "10%", width: "7%", height: "80%" }}
      data-tooltip="Ageuta"
    />
    <div
      className="shodan-map__item"
      style={{ left: "17%", width: "2%", height: "10%" }}
      data-tooltip="Ashirai"
    />
    <div
      className="shodan-map__item"
      style={{ left: "19%", width: "2%", height: "10%" }}
      data-tooltip="Yobikake"
    />
    <div
      className="shodan-map__item"
      style={{ left: "21%", width: "5%", height: "15%" }}
      data-tooltip="Yobikake"
    />
    <div
      className="shodan-map__item"
      style={{ left: "26%", width: "7%", height: "75%" }}
      data-tooltip="Mondo"
    />
    <div
      className="shodan-map__item"
      style={{ left: "33%", width: "2%", height: "40%" }}
      data-tooltip="Ageuta"
    />
    <div
      className="shodan-map__item"
      style={{ left: "35%", width: "3%", height: "45%" }}
      data-tooltip="Kuri"
    />
    <div
      className="shodan-map__item"
      style={{ left: "38%", width: "22%", height: "85%" }}
      data-tooltip="Sashi"
    />
    <div
      className="shodan-map__item"
      style={{ left: "60%", width: "2%", height: "15%" }}
      data-tooltip="Iguse"
    />
    <div
      className="shodan-map__item"
      style={{ left: "62%", width: "4%", height: "75%" }}
      data-tooltip="Mondo"
    />
    <div
      className="shodan-map__item"
      style={{ left: "66%", width: "7%", height: "55%" }}
      data-tooltip="Ageuta"
    />
    <div
      className="shodan-map__item"
      style={{ left: "73%", width: "5%", height: "70%" }}
      data-tooltip="Notto"
    />
    <div
      className="shodan-map__item"
      style={{ left: "78%", width: "3%", height: "100%" }}
      data-tooltip="Noriji"
    />
    <div
      className="shodan-map__item"
      style={{ left: "81%", width: "1%", height: "90%" }}
      data-tooltip="Hayafue"
    />
    <div
      className="shodan-map__item"
      style={{ left: "82%", width: "2%", height: "100%" }}
      data-tooltip="Noriji"
    />
    <div
      className="shodan-map__item"
      style={{ left: "84%", width: "6%", height: "70%" }}
      data-tooltip="Maibataraki"
    />
    <div
      className="shodan-map__item"
      style={{ left: "90%", width: "1%", height: "35%" }}
      data-tooltip="Noriji"
    />
    <div
      className="shodan-map__item"
      style={{ left: "91%", width: "9%", height: "95%" }}
      data-tooltip="Kiri"
    />
  </div>
);

export default ShodanTimeline;
