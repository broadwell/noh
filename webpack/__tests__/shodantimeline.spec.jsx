import React from "react";
import { shallow } from "enzyme";
import ShodanTimeline from "../components/ShodanTimeline";

describe("<ShodanTimeline>", () => {
  const sections = [
    {
      name: "Noriji",
      left: "82%",
      width: "2%",
      height: "100%"
    },
    {
      name: "Maibataraki",
      left: "84%",
      width: "6%",
      height: "70%"
    }
  ];
  it("renders as expected", () => {
    const component = shallow(<ShodanTimeline sections={sections} />);
    expect(component).toMatchSnapshot();
  });
});
