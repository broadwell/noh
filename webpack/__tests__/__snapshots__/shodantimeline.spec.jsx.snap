// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ShodanTimeline> renders as expected 1`] = `
<div
  className="shodan-map"
>
   
  <ShodanTimelineBlock
    height="100%"
    key="Noriji"
    left="82%"
    name="Noriji"
    width="2%"
  />
  <ShodanTimelineBlock
    height="70%"
    key="Maibataraki"
    left="84%"
    name="Maibataraki"
    width="6%"
  />
</div>
`;
